src/main/java/com/conveyal/datatools/manager/models/Model.java

```diff
@@ -96,7 +96,14 @@ public void storeUser(Auth0UserProfile profile) {
     public void storeUser(String id) {
         userId = id;
         if (!Auth0Connection.authDisabled()) {
-            Auth0UserProfile profile = Auth0Users.getUserById(userId);
+            Auth0UserProfile profile = null;
+            // Try to fetch Auth0 user to store email address. This is surrounded by a try/catch because in the event of
+            // a failure we do not want to cause issues from this low-level operation.
+            try {
+                profile = Auth0Users.getUserById(userId);
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
I think it might be good to have a log statement here that issues a warning or info about not finding a profile.